app/models/comment.rb

```diff
@@ -18,12 +18,12 @@ class Comment < ApplicationRecord
   validates :user_id, presence: true
 
   after_create   :after_create_checks
-  after_save     :calculate_score
-  after_save     :bust_cache
+  after_save     :calculate_score_async
```

Comment:
Ideally this remains called `calculate_score` and inside you trigger something called `Comments::CalculateScoreJob.perform_later`, same for the others